recipes/climlab-sbm-convection/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "climlab-sbm-convection" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/climlab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0852a75dbe8c486a302efb02cf56b776863d48fd446f498702e46a0ea7e9dc90
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<311 or win]  
+  number: 0
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - meson-python                           # [build_platform != target_platform]
+    - ninja                                  # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - {{ compiler('fortran') }}
+  host:
+    - meson-python
+    - python
+    - numpy
+    - pip
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
```

If this works with numpy 2.0 you don't need this. (If not, please pin numpy<2 above.)

```suggestion
    - numpy
```